pkg/scheduler/objects/queue.go

```diff
@@ -1017,6 +1019,8 @@ func (sq *Queue) SetMaxResource(max *resources.Resource) {
 		zap.String("current max", sq.maxResource.String()),
 		zap.String("new max", max.String()))
 	sq.maxResource = max.Clone()
+	sq.updateGuaranteedResourceMetrics()
```

Comment:
We only set max so we should not set the guaranteed metric.

Reply:
Good catch. Removed